pyblish/plugin.py

```diff
@@ -1277,19 +1278,39 @@ def discover(type=None, regex=None, paths=None):
                 continue
 
             for plugin in plugins_from_module(module):
-                if plugin.__name__ in plugins:
-                    log.debug("Duplicate plug-in found: %s", plugin)
-                    continue
+                # Check for duplicate plugin names. This is to preserve
+                # backwards compatility.
+                allow_duplicates = eval(
```

Comment:
What's `eval` for?

I'd suggest..

```python
allow_duplicates = bool(os.getenv("PYBLISH_ALLOW_DUPLICATE_PLUGIN_NAMES"))
```

At the header of the module and..

```python
                    if not allow_duplicates and plugin.__name__ in plugin_names:
                        log.debug("Duplicate plug-in found: %s", plugin)
                        continue
```

As a change to the body.

Reply:
> What's eval for?

allow_duplicates = bool(os.getenv("PYBLISH_ALLOW_DUPLICATE_PLUGIN_NAMES")) does not evaluate the string to a boolean. For example bool("False") is True.

Reply:
Aah, ok, no that's fine. If the variable exists, no matter its value, then it's `True`. Common practice.